examples/source/1.components/amp-bind-recaptcha.html

```diff
@@ -0,0 +1,113 @@
+    <link rel="canonical" href="https://www.xyz.com" />
+    <!-- import `amp-bind` so recaptcha can have multiple states -->
+    <script
+      async
+      custom-element="amp-bind"
+      src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"
+    ></script>
+    <!-- recaptcha input used to verify user for `amp-form` -->
+    <script
+      async
+      custom-element="amp-form"
+      src="https://cdn.ampproject.org/v0/amp-form-0.1.js"
+    ></script>
+    <!-- stylesheet used to import refresh image for recaptcha -->
+    <link
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+      rel="stylesheet"
+    />
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  </head>
+  <body>
+    <header>
+      <ol>
+        <li>Type anything in textbox below</li>
+        <li>Answer recaptcha equation correctly</li>
+        <li>Select submit</li>
+      </ol>
+      If you don't know the answer to the equation presented you can refresh the
+      recaptcha equation.
+    </header>
+    <!-- ## Implementation -->
+    <!-- recaptcha forces users to provide the correct input using the `[pattern]` requirement
+         The `[pattern]` updates dynamically as the state changes.
+         In order for the recaptcha to work on first pass, disable the input until the `amp-bind` `'state'` variable is set (i.e. `[disabled]="!state"`)
+        as another workaround, you could set a default pattern (without brackets) and have the `[pattern]` update as user refreshes the conditional.
+        Upon refresh the `'state'` is updated to provide a new equation.
+      -->
+    <div>
+    <amp-state id="captcha">
+      <script type="application/json">
+        {
+           "state1": {
+                 "result": "10",
+                 "condition": "+",
+                 "captchaCorrect": "6"
+
+           },
+           "state2": {
+                 "result": "2",
+                 "condition": "-",
+                 "captchaCorrect": "6"
+           },
+           "state3": {
+                 "result": "8",
+                 "condition": "*",
+                 "captchaCorrect": "2"
+           }
+         }
+      </script>
+    </amp-state>
+    <form action="https://www.google.com/" method="get" target="_top">
```

Comment:
also add a comment/explanation before form so that it'll show up in the doc page.

Reply:
updated